aimmo-game/simulation/avatar/avatar_wrapper.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, player_id, initial_location, worker_url, avatar_appearance):
         self.health = 5
         self.score = 0
         self.events = []
+        self.holdingTote = False
```

Comment:
not sure how that system will work, wouldn't it be better to have a collection of gathered pickups?

Reply:
I thought our idea was to have a single pickup and deliver it somewhere - but then that doesn't make the code very scalable if we want to change it, that's true.

Do we want to have multiple delivery items held at some particular one time?

Reply:
I guess just having a set of how many items of which type we have at one moment would work, here that would just be a set of one tote. That would be more scalable, even if this would be easy to refactor. What do you think @OlafSzmidt @mrniket ?